examples/create_wallet.rs

```diff
@@ -0,0 +1,20 @@
+use helium_wallet::wallet::Wallet;
+use std::path::Path;
+
+fn main() {
+    let password = "pass123";
+    let filename = Path::new("my-example-wallet.key");
+
+    let wallet = Wallet::builder()
+        .password(password)
+        .output(&filename)
+        .force(true)
+        .create()
+        .expect("it should have created a wallet");
+
+    let keypair = wallet
+        .decrypt(password.as_bytes())
```

Comment:
any way to use a trait for the `decrypt` argument so you don't have to as_bytes the password?

Reply:
That is the current implementation throughout the code base now. I would suggest, in another PR, we change all occurrences to `&str`. The get password call is returning a string anyways. I think a trait might be a bit overkill since we will likely only ever use a `String` as password.